cmd/detect.go

```diff
@@ -109,24 +134,70 @@ var detectCmd = &cobra.Command{
 				Rows:    sensorRows,
 			}
 
-			tables := []table.Table{fanTable, sensorTable}
+			printTables([]table.Table{fanTable, sensorTable})
+		}
+
+		nvControllers := nvidia.GetDevices()
+
+		if len(nvControllers) > 0 {
+			ui.Println("=========== nvidia: ===========\n")
+		}
+
+		for _, ctrl := range nvControllers {
 
-			for idx, table := range tables {
-				if table.Rows == nil {
-					continue
+			if len(ctrl.Fans) <= 0 && len(ctrl.Sensors) <= 0 {
+				continue
+			}
+
+			ui.Printfln("> Device: %s", ctrl.Identifier)
+			// TODO: print a label for the device from device.GetName() (e.g. "NVIDIA GeForce RTX 3060 Ti")?
```

Comment:
do you intend to implement this in this PR?

Reply:
I could, it's trivial after all, I'm just not sure how to format it there (to avoid giving the impression that the label can be matched)

Reply:
what do you think about
```
=========== nvidia: ===========

> Device: nvidia-10DE2489-0800
    Name: NVIDIA GeForce RTX 3060 Ti
  Fans     Index  Label  PWM  RPM   Auto
           1      Fan 1  30   1101  true
           2      Fan 2  31   1135  true
  Sensors  Index  Label        Value
           1      Temperature  33000
```
?

Reply:
Sure, its a start.